### PR TITLE
Revamp ConfigMap rollout tutorial

### PR DESCRIPTION
**Pull request by hausdorff**

### CODE REVIEW

File: quickstart/kubernetes/tutorial-configmap-rollout.md [hunk 1]
```diff
@@ -1,154 +1,223 @@
...
-title: "Tutorial: Graceful app rollout"
+title: "Tutorial: Getting Started With Pulumi"
 ---
 
-In this tutorial, we'll use nginx to reverse-proxy traffic to `pulumi.github.io`. The nginx
-configuration is contained in the file `default.conf` in this directory; this program reads that
-file and puts it in a `ConfigMap`. Hence, changing data in that file will cause register as a change
-in the `ConfigMap`'s data, which will trigger a rollout of the nginx `Deployment`.
+This tutorial shows you how to use Pulumi to write and manage a Kubernetes application through it's
+lifecycle. We will use the the Pulumi Kubernetes [TypeScript][ts] (an optionally-typed superset of
```
Comment (lukehoban) [inline quickstart/kubernetes/tutorial-configmap-rollout.md]: "the the" and other grammar oddness.

File: quickstart/kubernetes/tutorial-configmap-rollout.md [hunk 2]
```diff
@@ -1,154 +1,223 @@
 ---
-title: "Tutorial: Graceful app rollout"
+title: "Tutorial: Getting Started With Pulumi"
```
Comment (lukehoban) [inline quickstart/kubernetes/tutorial-configmap-rollout.md]: Does this new title make sense in context of where this material is linked?  I recall some inbound links might position it differently?

File: quickstart/kubernetes/tutorial-configmap-rollout.md [hunk 3]
```diff
@@ -1,154 +1,223 @@
...
+    The output should look something like this.
+
+    ![configmapRollout](/images/quickstart/kubernetes/cm-rollout.gif "ConfigMap-induced Rollout")
+
+2. **Verify the rollout worked.** You can do this by pasting the URL into the browser (be sure to
+   disable the cache), or by running the following:
+    ```sh
+    $ curl -sL $(pulumi stack output frontendIp) | grep -o "<title>Google</title>"
+    <title>Google</title>
+    ```
```
Comment (lukehoban) [inline quickstart/kubernetes/tutorial-configmap-rollout.md]: Do we need something about `pulumi destroy`?

File: quickstart/kubernetes/tutorial-configmap-rollout.md (hunk 2 above)
Comment (hausdorff) [reply quickstart/kubernetes/tutorial-configmap-rollout.md]: This should accurately reflect what people are coming here for.

**Pull request merged**